test/Extensions/ExpectedExceptionAttribute.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
```

Comment:
As of NUnit3, the `[ExpectedException]` attribute has been removed. However, the NUnit maintainers provided the source for it in case it was too challenging to remove it from a codebase. I opted to add it back to the project here.

Reply:
If you look at api, the test were rewritten in a few lines. Suggest having a look, took not long there